RUST/nstream-iter/src/main.rs

```diff
@@ -0,0 +1,174 @@
+///
+/// Copyright (c) 2020, Intel Corporation
```

Comment:
Do you want to add the appropriate copyright here?

Reply:
What's the convention for Copyright lines? Add or replace?

Reply:
New line please.